.env

```
@@ -25,3 +25,5 @@ FAVICON_URL=''
 ENABLE_LEARNER_RECORD_MFE=''
 LEARNER_RECORD_MFE_BASE_URL=''
 COLLECT_YEAR_OF_BIRTH=true
+APP_ID=
+MFE_CONFIG_API_URL=
```

I suggest using single quotes here. The effect is the same, but it would make these new lines consistent with the rest of the file, which could be less confusing for someone who doesn't know whether or not the quotes are important.
```suggestion
APP_ID=''
MFE_CONFIG_API_URL=''
```